front/app/containers/EventsPage/messages.ts

```diff
@@ -30,4 +30,8 @@ export default defineMessages({
     id: 'app.containers.eventspage.filterDropdownTitle',
     defaultMessage: 'Projects',
   },
+  errorWhenFetchingEvents: {
+    id: 'app.containers.eventspage.errorWhenFetchingEvents',
+    defaultMessage: 'Error when fetching events',
```

Comment:
I'd go for different copy. Something like "There was an error loading events. Please try reloading the page." Or something similar. You can ask Julienne if needed. She normally wouldn't see this copy when reviewing